src/fabric/src/fabric_doc_update.erl

```diff
@@ -106,11 +106,13 @@ before_doc_update(DbName, Docs, Opts) ->
         {true, _} ->
             %% fake db is expensive to create so we only do it if we have to
             Db = fabric_util:fake_db(DbName, Opts),
-            [couch_replicator_docs:before_doc_update(Doc, Db) || Doc <- Docs];
+            [couch_replicator_docs:before_doc_update(Doc, Db, interactive_updates)
```

Comment:
Hmm. I couldn't find where `interactive_updates` is handled. I think it is typo and should be `interactive_edit`.

Reply:
Good catch. Because this is for placeholder and I directly use https://github.com/apache/couchdb/pull/1808#discussion_r242498749. I already change it to replicated_changes.